### PR TITLE
:running: update the demo movie to 2.0.1 release

### PR DESCRIPTION
**Pull request by droot**
Two changes in this PR:
 - Updated the demo movie on README
 - Added instructions to generate demo movie

Note: Do not merge this as I am yet to test the svg rendering in README.md.

### CODE REVIEW

**Comment by mengqiy** [comment]
It seems it renders fine.

File: scripts/demo/README.md [hunk 1]
```diff
@@ -8,3 +8,25 @@ cd /tmp/kb-demo
...
+asciinema rec
+<path-to-KB-repo>/scripts/demo/run.sh
+
+<CTRL-C> to terminate the script
+<CTRL-D> to terminate the asciinema recording
+<CTRL-C> to save the recording locally
+
+# Edit the recorded file by editing the controller-gen path
+# Once you are happy with the recording, use svg-term program to generate the
+svg
```
Comment (DirectXMan12) [inline scripts/demo/README.md]: something wrapped here that shouldn't have

Comment (droot) [reply scripts/demo/README.md]: Fixed it.

**Comment by droot** [comment]
@mengqiy I checked https://github.com/droot/kubebuilder/tree/running/update-demo-movie in my tree and seems to render fine.

**Pull request merged**